code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -281,6 +281,25 @@
 			if(91.01 to INFINITY)
 				msg += "[t_He] [t_is] a shitfaced, slobbering wreck.\n"
 
+	if(has_trait(TRAIT_EMPATH) && !appears_dead && (src != user))

```

Comment:
```suggestion
	if(src != user && user.has_trait(TRAIT_EMPATH) && !appears_dead)
```
It should check if the user has the quirk, no?

Reply:
I believe so, strange how it worked in testing.